### PR TITLE
Document new requestId setup option

### PR DESCRIPTION
**Pull request by jem41**
This PR documents the new `requestId` setup option introduced in [#390](https://github.com/STORIS/mvom/pull/390).

### CODE REVIEW

File: website/docs/02 - Connection.md [hunk 1]
```diff
@@ -27,14 +27,15 @@ Connection.createConnection(mvisUri: string, account: string, options?: CreateCo
...
+| `maxReturnPayloadSize` | `number`      | `100_000_000`           | The maximum allowed return payload size in bytes. If this size is exceeded a DbServerError will be thrown. Returning large payloads can have a significant impact on performance and is often the result of invalid database records or an improperly configured query; ie forgetting to use pagination. Tune this value to match your datasets and the type of queries you issue. This can also be configured on a per request basis. |
+| `requestId`            | `string`      | randomly generated UUID | A request/trace ID to be passed to MVIS as a request header with the key `X-MVIS-Trace-Id`                                                                                                                                                                                                                                                                                                                                             |
```
Comment (shawnmcknight) [inline website/docs/02 - Connection.md:38]: This seems to be documented in the wrong place.  For `Connection` it looks like `open` was modified to accept options, but `createConnection` wasn't changed.

**Review by shawnmcknight — CHANGES_REQUESTED** [review]
Looks good.  Just the one note about the `Connection` docs having the `requestId` in the wrong place.

File: website/docs/02 - Connection.md (hunk 1 above)
Comment (jem41) [reply website/docs/02 - Connection.md]: I also missed the `getDbServerInfo` method documentation updates. Connection.md should be good now.

File: website/docs/02 - Connection.md [hunk 2]
```diff
@@ -136,29 +149,65 @@ Using the connection instance, you can access the database server's current date
...
 ```ts
-getDbDate(): Promise<string>
+getDbDate(options?: GetDbDateOptions): Promise<string>
 ```
 
+### Parameters
+
+| Parameter | Type     | Description                                              | Example |
+| --------- | -------- | -------------------------------------------------------- | ------- |
+| `options` | `object` | [Options object](#options-object-properties) (see below) |         |
```
Comment (shawnmcknight) [inline website/docs/02 - Connection.md:159]: With the links you need to be careful if the same heading appears in the document multiple times.  It won't go to the correct subheading.  If you look at the "Reading by Id" document you should be able to see how the link gets tweaked there to bring to the correct subheading.

Side note, if you use the "Open markdown preview" option in vscode you can test the links to make sure they bring you to the right place.

Comment (jem41) [reply website/docs/02 - Connection.md]: Links fixed.

File: website/docs/02 - Connection.md [hunk 3]
```diff
@@ -55,21 +55,34 @@ After a `Connection` instance has been created, the connection must be opened be
...
 import { Connection } from 'mvom';
 
 const mvisUri = 'http://foo.bar.com';
 const account = 'demo';
-const options = { timeout: 30_000 };
+const connectOptions = { timeout: 30_000 };
+const openOptions = { requestId: 'trace' };
 
 const makeConnection = async (): Connection => {
   const connection = Connection.createConnection(mvisUri, account, options);
```
Comment (kthompson23) [inline website/docs/02 - Connection.md:84]: `options` is now undefined in this example.

**Review by kthompson23 — CHANGES_REQUESTED** [review]
@jem41 Looks good but there's an error in the connection example now.

File: website/docs/02 - Connection.md (hunk 2 above)
Comment (shawnmcknight) [reply website/docs/02 - Connection.md]: Looks good!  Resolving this conversation.

File: website/docs/02 - Connection.md (hunk 3 above)
Comment (jem41) [reply website/docs/02 - Connection.md]: Fixed now

**Review by kthompson23 — APPROVED** [review]
Looks good, thank you.

**Review by shawnmcknight — APPROVED** [review]
LGTM!

**Pull request merged**